lib/liquid/array.rb

```diff
@@ -0,0 +1,15 @@
+module Liquid
+  module Array
+    def empty(input)
+      input.empty?
+    end
+
+    def count(input)
```

Comment:
Why not use the [`size` filter](https://github.com/Shopify/liquid/blob/e9b649b3455c63859f1b865a8684607d6ff5b050/lib/liquid/standardfilters.rb#L24)?